### PR TITLE
fill in the DL terms and most of the DB terms

### PR DESCRIPTION
**Pull request by crystina-z**

### CODE REVIEW

File: chinese-technical-terms.md [hunk 1]
```diff
@@ -88,10 +88,10 @@ perceptron | 感知器 | 感知器 | gǎnzhī qì
 multi-layer perceptron | 多层感知器 | 多層感知器 | duōcéng gǎnzhī qì
 word embedding | 单词嵌入 | 單詞嵌入 | dāncí qiànrù
 convolutional (neural) network | 卷积(神经)网络 | 卷積(神經)網絡 | juàn jī (shénjīng) wǎngluò
-feature map |
-convolutional filter |
-stride |
-(max/min) pooling |
+feature map | 特征图 | 特徵圖 | tèzhēng tú
+convolutional filter | 卷积滤波器 | 卷積過濾器 | juàn jī guòlǜ qì
+stride | 步长 ｜ 步長 | bù zhǎng
```
Comment (MXueguang) [inline chinese-technical-terms.md:93]: I think it should be ( bù cháng)?

btw, notice the `|` (en keybord) and `|` (zh keybord) are different. make sure you are typing '|' with English keybord layout, other wise there will be error in format.

File: chinese-technical-terms.md [hunk 2]
```diff
@@ -88,10 +88,10 @@ perceptron | 感知器 | 感知器 | gǎnzhī qì
...
+(max/min) pooling | (最大/最小) 池化 ｜(最大/ 最小) 池化 | (zuìdà / zuìxiǎo) chí huà
```
Comment (MXueguang) [inline chinese-technical-terms.md:94]: `(最大/最小) 池化 ｜(最大/ 最小) ` same here. use en keybord

File: chinese-technical-terms.md [hunk 3]
```diff
@@ -117,13 +117,13 @@ key-value store | 键值(数据)库 | 鍵值(數據)庫 | jiàn zhí (shùjù) k
...
-cluster |
-node |
-rack |
-index |
-query plan |
-logical plan |
-physical plan |
+API | 应用编程接口 | 應用編程接口 | yìngyòng biānchéng jiēkǒu 
+cluster | 集群 | 集群 | jíqún 
+node | 节点 ｜節點 | jiédiǎn
```
Comment (MXueguang) [inline chinese-technical-terms.md:122]: and here

File: chinese-technical-terms.md [hunk 4]
```diff
@@ -117,13 +117,13 @@ key-value store | 键值(数据)库 | 鍵值(數據)庫 | jiàn zhí (shùjù) k
...
+rack | 机架 | 機架 | jī jià
+index | 索引 | 索引 | suǒyǐn 
+query plan |
```
Comment (MXueguang) [inline chinese-technical-terms.md:125]: I am not sure, 查询计划？

**Review by MXueguang — CHANGES_REQUESTED** [review]

**Comment by lintool** [comment]
@crystina-z would you mind tweaking? ;) Thanks!

File: chinese-technical-terms.md [hunk 5]
```diff
@@ -49,12 +49,12 @@ validation set | 验证集 | 驗證集 | yànzhèng jí
...
+step size | 步长 | 步長 | bùcháng
 epoch | 轮 | 輪 | lún
 batch size | 批量样本数 | 批量樣本數 | pīliàng yàngběn shù
-learning rate | 学习率 | 學習率 | xuéxí lǜ 
-overfitting | 过拟合 | 過擬合 | guò nǐhé 
-under fitting | 欠拟合 | 欠擬合 | qiàn nǐhé 
+learning rate | 学习率 | 學習率 | xuéxí lǜ
+overfitting | 过拟合 | 過擬合 | guò nǐhé
+under fitting | 欠拟合 | 欠擬合 | qiàn nǐhé
 feed (i.e., to feed into a model) | 餵 | 餵 | wèi
```
Comment (MXueguang) [inline chinese-technical-terms.md:58]: sorry I missed this last time
“喂”

**Review by MXueguang — COMMENTED** [review]
The rest of the translations LGTM

File: chinese-technical-terms.md (hunk 5 above)
Comment (crystina-z) [reply chinese-technical-terms.md:58]: oh I was not checking the previous ones, thanks!

**Review by MXueguang — APPROVED** [review]

**Comment by lintool** [comment]
Thanks @crystina-z and @MXueguang !

**Pull request merged**